discovery-provider/alembic/versions/ed974e76d415_fix_indexing_track_slug.py

```diff
@@ -0,0 +1,414 @@
+                "mood": True,
+                "tags": True,
+                "share": True,
+                "play_count": True,
+                "remixes": True,
+            },
+            stem_of=None,
+        ),
+        # trackId - 470911, blocknumber - 22477832
+        Track(
+            track_id=470911,
+            blocknumber=22477832,
+            is_current=True,
+            is_delete=False,
+            txhash="0xda0b64477469390eaa3b17098c01f3e03e5b0e52959ffd493804538dda039a92",
+            blockhash="0x705d01ed50cf8f1d83686acc19c9d0bdd52493b3552532543629cdf07fba6005",
+            owner_id=3538,
+            route_id="",
+            cover_art=None,
+            cover_art_sizes="QmZFgQiupP2KWoqYZGqJ83zEwHtXMj6QhnNNfDCLpTax4q",
+            title="???",
+            tags="electro,house,2021",
+            genre="Electro",
+            mood="Fiery",
+            credits_splits=None,
+            remix_of=None,
+            create_date="",  # TODO: get date
+            release_date="Fri Jul 30 2021 17:29:24 GMT+0200",  # TODO: check if converts
+            length=None,
+            file_type=None,
+            description=None,
+            license="All rights reserved",
+            isrc=None,
+            iswc=None,
+            track_segments=[
+                {
+                    "multihash": "QmW7kQwqbZcYWQHtYdXK6KezSCqQ1vRdwhdWtWvTT79rnk",
+                    "duration": 6.016,
+                },
+                {
+                    "multihash": "QmYQEkQvPGHgkMLGhvfYcZdd385KmoifAer5kxnAuYhf4E",
+                    "duration": 5.994667,
+                },
+                {
+                    "multihash": "QmVj5uvFPBpa9635hL17EcYPJAQURm6rqJdvociR6Qk1Q2",
+                    "duration": 5.994667,
+                },
+                {
+                    "multihash": "QmW5QyPo5kUSXs1v5oJTrLWdFB1JHciWKcMpw3GbY7a7rX",
+                    "duration": 5.994667,
+                },
+                {
+                    "multihash": "QmZeRjqj9SC1depFGw5vLwdAiayBr7rdrtSiUKYtoFaJix",
+                    "duration": 6,
+                },
+                {
+                    "multihash": "QmQ2dPUnNMay2Yxj6pj7Jw7WvpodXoQxZvUTdSKjMcoF56",
+                    "duration": 0.021333,
+                },
+            ],
+            download={
+                "is_downloadable": False,
+                "requires_follow": False,
+                "cid": None,
+            },
+            updated_at="July-30-2021 11:30:05",
+            created_at="July-30-2021 11:30:05",
+            is_unlisted=False,
+            field_visibility={
+                "genre": True,
+                "mood": True,
+                "tags": True,
+                "share": True,
+                "play_count": True,
+                "remixes": True,
+            },
+            stem_of=None,
+        ),
+    ]
+    session.add_all(tracks)
+
+    track_routes = [
+        TrackRoute(
+            slug="oWXPm",
+            title_slug="oWXPm",
```

Comment:
@raymondjacobson Can you verify the track routes are correct

Reply:
yes these are correct!